frontend/src/views/web/challenge/my-submission.html

```diff
@@ -1,7 +1,7 @@
 <section ng-if="challenge.isParticipated" class="ev-sm-container ev-view ">
     <div class="row">
         <div class="col s12 m12">
-            <div class=" ev-card-panel ev-z-depth-5 ">
+            <div class="ev-challenge-view ev-card-panel ev-z-depth-5 ">
```

Comment:
I don't think we need this class here? Please check again @NSDCars5

Reply:
The class was providing the bottom margin to separate the two cards.

With the class:

![localhost_8888_web_challenges_challenge-page_1_my-submission laptop](https://user-images.githubusercontent.com/5107795/34906474-7e1d8cde-f894-11e7-89e5-7fbc37b69d60.png)

Without it:

![localhost_8888_web_challenges_challenge-page_1_my-submission laptop 1](https://user-images.githubusercontent.com/5107795/34906477-84800a3e-f894-11e7-8bf5-4c50b0b2a62b.png)

Should I remove it and let the cards come together? This same class is used in the top cards of all other tabs where there is more than one card.